src/main/content/_assets/js/collections.js

```diff
@@ -16,24 +15,46 @@ $(document).ready(function() {
     });
 });
 
+function handleCollectionsRequests(instanceName){
+    getCollectionData(instanceName);
+    getCliVersion(instanceName);
+}
+
 function getCollectionData(instanceName){
     if(typeof instanceName === "undefined"){
         return;
     }
-
     return fetch(`/api/auth/kabanero/${instanceName}/collections/list`)
         .then(function(response) {
             return response.json();
         })
+        .then(function(data){
+            return updateCollectionView(data);
+        })
```

Comment:
this can probably just be 

```
        .then(updateCollectionView)
```

same with 
```
        .then(function(data){
            return setCLIVersion(data);
        })
```